### ISSUE TITLE
openUrl GET request instead POST and vice versa

### ISSUE THREAD

**Issue by Kirill89** [laurentj/slimerjs#362]
Looks like `httpConf` parameter of `webpage.openUrl` is broken. Then I set it to `{ operation: 'GET', data: {} }` - openUrl make POST request. Then I set it to `{ operation: 'POST' }` - openUrl make GET request.

I think `operation` key ignored and request method depend only on `data`.

Demo: https://github.com/Kirill89/slimer-test

**Comment by laurentj** [comment]
Yes, get or post depends only if data is set or not. I should improve that.
